.github/workflows/spelling-check-bot.yml

```
@@ -0,0 +1,51 @@
+name: Create an issue to report typos
+
+on:
+  schedule:
+    - cron: "0 0 * * mon"
+  workflow_dispatch:
+
+jobs:
+  sync:
+    if: github.repository == 'mdn/content'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
```

We have this step twice, is that right? Should we move the full checkout step here and delete the one below?

```suggestion
      - uses: actions/checkout@v4
        with:
          repository: mdn/content
          ref: main
          path: content
```